app/views/greetings/hello.html.erb

```diff
@@ -1,17 +1,12 @@
 <head>
+  <title>Home - Wye RetroApp</title>
```

Comment:
I think that this is not working. Maybe you can make use of [a named yield](https://guides.rubyonrails.org/layouts_and_rendering.html#understanding-yield) and the [content_for](https://guides.rubyonrails.org/layouts_and_rendering.html#using-the-content-for-method) helper to achieve this objective.
[This question](https://stackoverflow.com/questions/185965/rails-how-to-change-the-title-of-a-page) can help too!

Reply:
That's correct :D this is fixed in https://github.com/anita040593/RetroApp/pull/3 (hopefully)

But since it's not working in this PR I'll delete the line :)